dbms/src/TiDB/Schema/TiDBSchemaSyncer.h

```diff
@@ -110,7 +115,7 @@ struct TiDBSchemaSyncer : public SchemaSyncer
         Stopwatch watch;
         SCOPE_EXIT({ GET_METRIC(tiflash_schema_apply_duration_seconds).Observe(watch.elapsedSeconds()); });
 
-        LOG_INFO(log, "Start to sync schemas. current version is: {} and try to sync schema version to: {}", cur_version, version);
+        LOG_INFO(log, "[keyspace {}] Start to sync schemas. current version is: {} and try to sync schema version to: {}", keyspace_id, cur_version, version);
```

Comment:
Change the type of `log` to be `DB::LoggerPtr` and use `auto ks_log = log->getChild(fmt::format("keyspace={}", keyspace_id))`. Using it can add a suffix in the logging.